README.md

```diff
@@ -1 +1,169 @@
+    <li><a href="#">REACT JS </a></li>
+
+    <li><a href="#">JSX</a></li>
+     <li><a href="#">Js</a></li>
+  </ul>
+</details>
+
+
+
+
+
+### Key Features <a name="key-features"></a>
+
+
+- **[Display a list of Books]**
+- **[Add a book to the list  ]**
+- **[Remove the Book form the list ]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+[N/A]
+
+
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+- In order to run this project you need:
+
+- [git](https://git-scm.com/downloads): A tool for managing source code
+- [Visual Studio Code](https://code.visualstudio.com/): A source code editor
+- [Live Server](https://marketplace.visualstudio.com/items?itemName=ritwickdey.LiveServer): A simple web server utility for Visual Studio Code
+
+### Setup
+
+Clone this repository to your desired folder:
+ https://github.com/josuerushanika/Bookstore.git
+
+ Example commands:
+
+```sh
+  cd my-folder
+  git clone https://github.com/josuerushanika/Bookstore.git
+```
+
+### Install
+
+Install this project with:
+
+` cd my-project
+  - Run npm install after cloning the repo
+
+### Usage
+
+### Usage
+
+- Run npm start for development. 
+- Run npm run build for production
+
+
+### Run tests
+
+To run tests, run the following command:
+` npm run test`
+
+
+### Deployment
+
+You can deploy this project using:
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Author1**
+
+- GitHub: [@josue_Rushanika]( https://github.com/josuerushanika)
+- Twitter: [@josue_Rushanika]( https://twitter.com/josuerushanika)
+- LinkedIn: [josue_Rushanika](https://www.linkedin.com/in/josue-rushanika-336049b3/)
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[Full navigation of the page ]**
+- [ ] **[Improve the UI/UX design  ]**
+- [ ] **[Add more Features of filtering the books ]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project please give it a start and share it with your friends. 
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to appreciate[Microverse](https://www.microverse.org/) for providing the resources and the platform for us to be a Professional full-stack developer.
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] @josuerushanika kindly ensure your license link is working correctly, currently it's broken.